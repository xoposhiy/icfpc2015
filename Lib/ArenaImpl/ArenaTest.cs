using System.IO;
using ApprovalTests;
using Lib.Intelligence;
using Newtonsoft.Json;
using NUnit.Framework;

namespace Lib.ArenaImpl
{
    [TestFixture]
    public class ArenaTest
    {
        [Test]
        public void Test()
        {
            var arena = new Arena();
//            var solver = new PhrasesOnlySolver();
//            var solver = new NamiraOracle();
            var solver = new AzuraOracle();
            var res = arena.RunAllProblems(solver);
//            File.WriteAllText("arena.json", JsonConvert.SerializeObject(res, Formatting.Indented));
            Approvals.Verify(res);
        }
    }
}