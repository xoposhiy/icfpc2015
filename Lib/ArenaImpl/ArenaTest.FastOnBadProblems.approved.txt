e
Adaptive
TotalScore: 15316
Id	Perf	Avg	ScoreBySeeds
4	126%	15316	15316